.travis.yml

```diff
@@ -19,7 +19,9 @@ matrix:
         - php: hhvm
 
 before_install:
+    - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
     - composer self-update
+    - if [ "$GITHUB_OAUTH_TOKEN" ]; then composer config --global github-oauth.github.com ${GITHUB_OAUTH_TOKEN}; fi;
```

Comment:
https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings

> Similarly, we do not provide these values to untrusted builds, triggered by pull requests from another repository.

:disappointed:

Looks like we can't force this, unless we have a fallback token, like https://github.com/Sylius/Sylius/blob/3d9528532d20b82d56d8e4bd1eadfa3695f1f486/.travis.yml#L42-L43